### ISSUE TITLE
Issue with Application ID

### ISSUE THREAD

**Issue by dkmea** [Romanitho/WingetIntunePackager#29]
Hi,

Recently I am starting to face an issue where I cannot connect to any tenants.
I get following error:
"Application with identifier 'd1ddf0e4-d672-4dae-b554-9d5bdfd93547' was not found in the directory"

It seems to be related to the identifier being deprecated, reading following article:
https://learn.microsoft.com/en-us/answers/questions/1659638/weve-detected-a-microsoft-intune-powershell-script

**Comment by spxnstevens** [comment]
Also interested in this update as we can no longer use the tool.

**Comment by stuman333** [comment]
Same issue here great tool, but currently broken....... :-(

**Comment by FullDad1** [comment]
Yeah they changed something a few weeks ago that broke this.  Just needs to be repackaged.  Would love to have this back.  Saves me so much time.

**Comment by Romanitho** [comment]
Hi guys. Not had time yet to work on that. Very busy :/

**Comment by AdamJParker** [comment]
Any update on this? If there is a work around to get it to connect to the Tenant please let me know.

**Comment by Romanitho** [comment]
Sorry, no update right now. I need to take time to work on that, but not really my priority at the moment :/

**Comment by LunianDE** [comment]
> Sorry, no update right now. I need to take time to work on that, but not really my priority at the moment :/

@Romanitho I created a PR to support Microsofts Change :)

**Comment by kylejericson** [comment]
Anyupdate on this?

**Comment by LunianDE** [comment]
> Anyupdate on this?
@stuman333 

Its fixed :) See #30  for Documentation on how to set it up. Just download the new PowerShell Script in the Sources Folder and fill out the Text Boxes. 

Romanitho didn't make a Release yet, but downloading it from the Repo itself should work i think

**Comment by huuub** [comment]
As I get it right it is not working at the moment? 
I still can't connect. I get a login screen but it just closes. 
I created the app and gave the right privs and the correct redirect url.
I think in the readme should be noted that you need to make this app registration in Azure.
